src/sdc11073/pysoap/soapclient.py

```diff
@@ -387,6 +363,12 @@ def _make_get_headers(self) -> dict[str, str]:
 
     def get_from_url(self, url: str, msg: str) -> bytes:
```

Comment:
why there isnt such error handling in this method like in `_send_soap_request`

Reply:
Error handling can be delegated to caller, because this method is never called in a background thread. 
_send_soap_request is called in threads, and this means logging of error, closing connection etc. must be done by sdc11073 somehow.